cfme/tests/services/test_dynamicdd_dialogelement.py

```diff
@@ -48,14 +49,14 @@ def dialog(copy_instance, create_method):
         'choose_type': "Drop Down List",
         'dynamic_chkbox': True
     }
-    dialog = ServiceDialog(label="dialog_" + fauxfactory.gen_alphanumeric(),
-                           description="my dialog", submit=True, cancel=True,
-                           tab_label="tab_" + fauxfactory.gen_alphanumeric(),
-                           tab_desc="my tab desc",
-                           box_label="box_" + fauxfactory.gen_alphanumeric(),
-                           box_desc="my box desc", element_data=element_data)
-    dialog.create()
-    yield dialog
+    sd = service_dialogs.create(label=dialog,
+        description="my dialog", submit=True, cancel=True,)
+    tab = sd.dialogtab.create(tab_label='tab_' + fauxfactory.gen_alphanumeric(),
+        tab_desc="my tab desc")
+    box = tab.dialogbox.create(box_label='box_' + fauxfactory.gen_alphanumeric(),
+        box_desc="my box desc")
+    box.dialog_element.create(element_data=[element_data])
+    yield sd
```

Comment:
You can probably add a helper method to the DialogCollection, that will create a simple dialog with signle element.

Something like:
```
simple_dialog = dialogs.create_simple('label', 'tabname', 'boxname', element_data)
```